### ISSUE TITLE
Consider using a different port of zxing

### ISSUE THREAD

**Issue by dandv** [miguelao/shapedetection-polyfill#1]
Not saying [it](https://github.com/edi9999/jsqrcode)'s better, but it looks [more responsibly maintained](https://github.com/jesstelford/jsqrcode/pull/3).

**Comment by dandv** [comment]
Yup - upon some light prodding, @jesstelford has just [deprecated the package](https://github.com/jesstelford/jsqrcode) (thanks!) in favor of @edi9999's.

**Comment by edi9999** [comment]
Thanks for using my port, I'm trying to maintain it, but I'm also searching for additional maintainers, so feel free to contact me if you would have interest to do that.

**Comment by yell0wd0g** [comment]
hey sorry for the delay @edi9999, I might be of help maintaining your port; perhaps we could consider extending support to other codes that ZXing supports and jsqrcode don't.  I might not have bags of time but I can make some.

**Comment by dsanders11** [comment]
@miguelao, gonna throw [my port of ZXing](https://github.com/dsanders11/barcode.js/tree/1D-barcodes) (1D-barcodes branch) into the ring as well. It's a fork of [Schibum/barcode.js](https://github.com/Schibum/barcode.js/) which was re-organized to more closely match ZXing's directory structure and begin to port 1D barcodes like Code 128. It's compiled using Closure and has lots of type-checking, decent test coverage, and is easy to develop. I haven't ported all 1D barcode support because I covered my needs, but it would only take a few hours to port the rest of the 1D barcodes from ZXing, it's just step and repeat using the existing pattern, and easy enough to plug in their test data to confirm functionality.
